client/components/AdoptionForm.js

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from "react"
+import _ from "lodash"
+
+const AdoptionForm = () => {
+  const [adoptionData, setAdoptionData] = useState({
+    name: "",
+    phoneNumber: "",
+    email: "",
+    homeStatus: "own"
+  })
+  let errorContainer = null
+  const [errors, setErrors] = useState([])
+  if (errors.length !== 0) {
+    errorContainer = errors.map(error => {
+      error = error === "phoneNumber" ? "phone number" : error
+      return <p>{`${_.capitalize(error)} is empty`}</p>
+    })
+  }
```

Comment:
I would just rearrange this slightly for readability and to keep the `useState` statements together 

```js
  const [adoptionData, setAdoptionData] = useState({
    name: "",
    phoneNumber: "",
    email: "",
    homeStatus: "own"
  })
  const [errors, setErrors] = useState([])
  
  let errorContainer = null
  if (errors.length !== 0) {
    errorContainer = errors.map(error => {
      error = error === "phoneNumber" ? "phone number" : error
      return <p>{`${_.capitalize(error)} is empty`}</p>
    })
  }
```